engine/src/main/java/io/graphqlcrud/Filters.java

```diff
@@ -0,0 +1,88 @@
+package io.graphqlcrud;
+
+import graphql.Scalars;
+import graphql.schema.*;
+
+public class Filters {
+
+    public static GraphQLInputObjectType.Builder pageInputBuilder() {
+        return GraphQLInputObjectType.newInputObject().name("PageRequest")
+                .field(GraphQLInputObjectField.newInputObjectField().name("limit").type(Scalars.GraphQLInt))
+                .field(GraphQLInputObjectField.newInputObjectField().name("offset").type(Scalars.GraphQLInt));
+    }
+
+    public static GraphQLInputObjectType.Builder filterInputBuilder() {
```

Comment:
Is this some form of sandbox. It contains business model where it should be generic?

Reply:
Based on the schema we read from the Database we are applying the where filters and page limits apply automatically.